### PR TITLE
Add Week 5 slides - functions, documentation, namespace part 1

### PR DESCRIPTION
**Pull request by jonleslie**
This will bring the files for my presentation on 2020-10-27.

### CODE REVIEW

**Comment by jonleslie** [comment]
I'm confused as to why Jon Harmon's previous commits are showing up in this pull request. Perhaps it has something to do with how I merged from the upstream to my local main branch? Please tell me if I've done something dumb and I I'll try to fix it.

File: Presentations/Week05/Cohort1/functions_docs_namespace.Rmd [hunk 1]
```diff
@@ -0,0 +1,627 @@
+---
```
Comment (tanho63) [inline Presentations/Week05/Cohort1/functions_docs_namespace.Rmd:1]: For consistency's sake, can you rename this folder from Cohort1 to Cohort01?

**Review by tanho63 — CHANGES_REQUESTED** [review]

**Comment by jonthegeek** [comment]
I set this up to merge into a dev branch. If we squash and merge, we can kill the weird merge history here, and then ~easily merge into main. I think.

**Comment by tanho63** [comment]
OK I think I did the thing.

**Pull request merged**